iron-image.html

```diff
@@ -396,8 +389,14 @@
       },
 
       _resolveSrc: function(testSrc) {
-        var baseURI = /** @type {string} */(this.ownerDocument.baseURI);
-        return (new URL(Polymer.ResolveUrl.resolveUrl(testSrc, baseURI), baseURI)).href;
+        var resolved = Polymer.ResolveUrl.resolveUrl(testSrc, this.$.baseURIAnchor.href);
```

Comment:
Looking at the prior implementation, this may not have been a problem or went unsolved before